azure-pipelines.yml

```diff
@@ -25,6 +25,13 @@ jobs:
       vmImage: ubuntu-latest
       container: jasperdm/alpine-watchman:latest
       arch: x64
+
+  - template: azure-pipelines-template.yml
```

Comment:
You'd also want a non-Alpine build, right? Same as for Linux/x64

Reply:
For now I'd be happy with an Alpine build, but feel free to add any code suggestions! Not sure why this repo has no activity since January.